### PR TITLE
Feat/tabability

### PR DESCRIPTION
**Pull request by LordSchwifty**
Added accessibility to the webApp. This gives the ability to tab through the website and navigate it without a mouse. In here we added tabindex to different html elements and gave the enter button functionality on scripts. Expecting a merge conflict possibly.

### CODE REVIEW

**Review by jsahim — APPROVED** [review]
Looks good! We'll review together before we resolve the merge conflict.

File: dist/index.html [hunk 1]
```diff
@@ -9,11 +9,11 @@
 </head>
 <body>
     <nav class="nav-bar">
-        <img class="nav-logo pointer" id="logo" src="./images/home-button.png" alt="Take Me Home">
+        <img class="nav-logo pointer" id="logo" src="./images/home-button.png" alt="Take Me Home" tabindex="0">
```
Comment (qrispi) [inline dist/index.html:12]: Love the tab-index. So much better than re-creating everything as a button!

File: src/scripts.js [hunk 2]
```diff
@@ -31,17 +31,21 @@ let currentView;
...
 searchBar.addEventListener('keyup', searchRecipes);
 
 //functions
 apiCalls().then(data => {
-  users = data[0].usersData;
-  ingredients = data[1].ingredientsData;
-  recipes = data[2].recipeData;
+  users = data[0].users;
+  ingredients = data[1].ingredients;
+  recipes = data[2].recipes;
```
Comment (qrispi) [inline src/scripts.js]: Those sneaky guys.... always keeping us on our toes!

File: src/scripts.js [hunk 3]
```diff
@@ -86,12 +90,14 @@ function createRecipeModal(event) {
         ${createList(modalRecipe.getInstructions())}
         </ol>
         <h4 class="oregano-font"><i>TOTAL COST $${+(modalRecipe.listCost(ingredients))}</i></h4>
-        <button class="save-button pointer" id="saveBtn">${updateButtonText()}</button>
+        <button class="save-button pointer" id="saveBtn" tabindex="0">${updateButtonText()}</button>
     </div>`;
+    document.querySelector('#closeIcon').focus()
```
Comment (qrispi) [inline src/scripts.js:107]: Throwback to Turdle!

File: dist/index.html (hunk 1 above)
Comment (jfogiato) [reply dist/index.html:12]: +1 - great fix here and a huge learning!

**Review by qrispi — APPROVED** [review]
This merge looks great but I wonder if we need to add any aria roles in the next push??

File: src/scripts.js [hunk 4]
```diff
@@ -31,17 +31,21 @@ let currentView;
 
 //event listeners
 recipeSection.addEventListener('click', createRecipeModal);
+recipeSection.addEventListener('keypress', createRecipeModal);
```
Comment (jfogiato) [inline src/scripts.js:34]: @LordSchwifty Does this allow _any_ key to open the modal? If so, do we want that functionality?

cc @jsahim @qrispi

**Review by jfogiato — APPROVED** [review]
Looks great - thanks for the work on this, y'all! ACCESSIBILITY KINGS

File: src/scripts.js [hunk 5]
```diff
@@ -31,17 +31,21 @@ let currentView;
...
 modalSection.addEventListener('click', collapseRecipe);
 filterDropdown.addEventListener('click', filterRecipes);
 navMyRecipes.addEventListener('click', showSavedRecipes);
+navMyRecipes.addEventListener('keypress', (event) => {
+    if(event.key === 'Enter') showSavedRecipes()});
 logo.addEventListener('click', goHome);
+logo.addEventListener('keypress', goHome);
```
Comment (qrispi) [inline src/scripts.js]: This piece needs this: "('keypress', (event) => {
    if(event.key === 'Enter') goHome()})" added to the event listener.

**Pull request merged**